pkg/scheduler/plugins/capacity/capacity.go

```diff
@@ -301,19 +301,12 @@ func (cp *capacityPlugin) OnSessionOpen(ssn *framework.Session) {
 		klog.V(5).Infof("job %s min resource <%s>, queue %s capability <%s> allocated <%s> inqueue <%s> elastic <%s>",
 			job.Name, minReq.String(), queue.Name, attr.realCapability.String(), attr.allocated.String(), attr.inqueue.String(), attr.elastic.String())
 		// The queue resource quota limit has not reached
-		r := minReq.Add(attr.allocated).Add(attr.inqueue).Sub(attr.elastic)
```

Comment:
Here should also ut, it's a bugfix here through git commit history.

Reply:
You means the pr https://github.com/volcano-sh/volcano/pull/2637？ If the resource in request doesn't exist in capability, it remove from it, that is strange. eg, the pods or ephemeral-storage exist in node's `Allocatable`, it also should be a resource in capability.

Reply:
Yeah, can the previous pr's problem be fixed?